src/main/java/Exceptions/UserAlreadyExistsException.java

```diff
@@ -0,0 +1,29 @@
+package Exceptions;
+
+import java.lang.RuntimeException;
+import org.springframework.http.HttpStatus;
+import Exceptions.APIException;
+
+public class UserAlreadyExistsException extends RuntimeException implements APIException {
+
+    private HttpStatus http_status = HttpStatus.CONFLICT;
+    private String message = "Username or Email-ID already exists";
```

Comment:
Can be more specific here whether it was `Username` or `Email ID` which was already existing